connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSourceTask.java

```diff
@@ -159,7 +159,7 @@ public void execute() {
                 }
 
                 if (toSend == null) {
-                    log.debug("{} Nothing to send to Kafka. Polling source for additional records", this);
+                    log.trace("{} Nothing to send to Kafka. Polling source for additional records", this);
```

Comment:
I'd rather not make this change. It is helpful, and if we change the level to trace, people may still expect to see it.

Reply:
They can see it on trace level, right. For people like me it creates gigabytes of useless logs on debug level that does not help.

Reply:
TRACE level seems appropriate since it's basically just a liveness message and I can imagine it being quite spammy.

Reply:
You can imagine it if you run connector and see tons of useless logs you need to remove just to see something else.

Reply:
Yeah, I'm fine with this.